### DISCUSSION TITLE
clickhouse backup data path

### DISCUSSION THREAD

**Discussion by honobi** [Altinity/clickhouse-backup#984] [Q&A] [answered]
I am using a containerized ClickHouse, and the ClickHouse data directory is located on an externally mounted filesystem (K8s PVC). When using clickhouse-backup to perform backups, the data is backed up to the container's /var/lib/clickhouse/backup/ directory. Can this backup data directory be modified?
If the data is backed up by default to the /var/lib/clickhouse/backup/ directory, will this result in a copy of the data files? (If they are on the same filesystem, ClickHouse's backup uses hard links and does not require copying the data.)

**Comment by Slach** [comment]
could you share
`SELECT * FROM system.disks`?

**Reply by Slach** [reply, marked as answer]
usually just mount your PVC into `/var/lib/clickhouse/` inside Pod

do you use clickhouse-operator?
